ros/api/v1/hosts.py

```diff
@@ -155,6 +162,7 @@ def get(self):
                 host['os'] = row.System.deserialize_host_os_data
                 host['report_date'] = row.PerformanceProfile.report_date
                 host['number_of_recommendations'] = row.PerformanceProfile.number_of_recommendations
+                host['groups'] = row.System.groups if row.System.groups else []
```

Comment:
@PreetiW,  as you defined the field as list above when groups is None it will still sent out empty array. No need to handle explicitly here.

In addition, you can extend `SYSTEM_COLUMNS` constant with `groups`key  [here](https://github.com/RedHatInsights/ros-backend/blob/a5bf36c65072032f47f6e46701011cad643cfaf8/ros/api/v1/hosts.py#L42),  it will add `groups` key into object at [line 156](https://github.com/RedHatInsights/ros-backend/blob/a5bf36c65072032f47f6e46701011cad643cfaf8/ros/api/v1/hosts.py#L156) and you can remove this line 165.

Reply:
Thank you for the suggestion @kgaikwad. I have updated the code and will push the changes in the latest commit. 🙌🏻